src/jvmMain/kotlin/com/sdercolin/vlabeler/ui/editor/EditorState.kt

```diff
@@ -186,7 +186,8 @@ class EditorState(
                 val updated = chartStore.prepareForNewLoading(project, appConf, it)
                 appState.updateProjectOnLoadedSample(it, moduleName)
                 if (updated) {
-                    val renderProgressTotal = it.chunkCount * (it.channels + if (it.hasSpectrogram) 1 else 0)
+                    val renderProgressTotal = it.chunkCount *
+                        (it.channels + (if (it.hasSpectrogram) 1 else 0) + (if (it.hasPower) it.powerChannels else 0))
```

Comment:
We need to update `totalCharts` in `ChartPrerenderer.kt L72` as well. Since the logic is the same, we can have a property in `SampleInfo` :

```kotlin
val SampleInfo.totalChartCount: Int get() = ...
```